search.py

```diff
@@ -33,37 +31,48 @@ def comparator(tup1, tup2):
         return tup2[1] - tup1[1]
 
 # Parsing
-
-def process(line):
-    """
-    Filters out some punctuations and then case-folds to lowercase
-    Takes in a String line and returns the result String
-    """
-    return filter_punctuations(line).lower()
-
 def filter_punctuations(s):
     """
     Replaces certain punctuations from Strings with space, to be removed later on
-    Takes in a String s and returns resultant String
+    Takes in String s and returns the processed version of it
     """
-    punctuations = '''!?-;:"\\,./#$%^&<>[]{}*_~()'''
+    punctuations = '''!?-;:\\,./#$%^&<>[]{}*_~()'''
```

Comment:
Have added more punctuation to remove in the index.py. Can copy over

Reply:
yea i should probably import it in also